base/services/w32time/ntpclient.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * PROJECT:     ReactOS Timedate Control Panel
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dll/cpl/timedate/ntpclient.c
+ * PURPOSE:     Queries the NTP server
+ * COPYRIGHT:   Copyright 2006 Ged Murphy <gedmurphy@gmail.com>
+ *
+ */
+
+#include "timedate.h"
+
+#include <winsock2.h>
+
+#define TIMEOUT 4000 /* 4 second timeout */
+
+typedef struct _INFO
+{
+    SOCKET Sock;
+    SOCKADDR_IN myAddr;
+    SOCKADDR_IN ntpAddr;
+    NTPPACKET SendPacket;
+    NTPPACKET RecvPacket;
+} INFO, *PINFO;
+
+
+static BOOL
+InitConnection(PINFO pInfo,
+               LPSTR lpAddress)
+{
+    WSADATA wsaData;
+    HOSTENT *he;
+    INT Ret;
+
+    Ret = WSAStartup(MAKEWORD(2, 2),
+                     &wsaData);
```

Comment:
Why is this on a newline? (You do this in 80% of the function calls, even when arguments are not that long...)

Reply:
This was copied exactly from dll/cpt/timedate/ntpclient.c by Ged Murphy as mentioned previously.